love/src/components/ESS/AuxTel/AuxTelESS.container.jsx

```diff
@@ -146,17 +185,22 @@ const parseHumidity = (streams) => {
   streams.forEach((stream) => {
     Object.entries(stream).forEach((entry) => {
       const essData = entry[1];
-      const sensorName = essData?.sensorName.value ?? '';
-      const value = essData?.relativeHumidity.value ?? 0;
-      const location = essData?.location.value ?? '';
+      const sensorName = essData?.sensorName?.value ?? '';
+      const value = essData?.relativeHumidity?.value ?? 0;
+      const location = essData?.location?.value ?? '';
+      const timestamp = essData?.timestamp?.value ?? undefined;
+      const xPosition = essData?.xPosition?.value ?? undefined;
+      const yPosition = essData?.yPosition?.value ?? undefined;
+      const zPosition = essData?.zPosition?.value ?? undefined;
```

Comment:
Same as suggested above.
```suggestion
      const timestamp = essData?.timestamp?.value;
      const xPosition = essData?.xPosition?.value;
      const yPosition = essData?.yPosition?.value;
      const zPosition = essData?.zPosition?.value;
```